site/_posts/2021-02-16-orbitvu-fusionauth-story.md

```diff
@@ -0,0 +1,88 @@
+
+Last but not least is that at Orbitvu we love open source and we use it a lot. What we do like in FusionAuth is that, while not open source, it feels like an open source project due to its development style with Github's issue tracker.
+
+> We’re integrating with FusionAuth using its API and it works flawlessly.
+
+**Dan:** I see the Orbitvu website supports multiple languages; was the ability to localize FusionAuth important as well?
+
+**Maciej:** Definitely, it was one of the must-have features when we were looking for our auth solution.
+
+**Dan:** How much time and money would you say FusionAuth has saved you?
+
+**Maciej:** Given that FusionAuth is the only solution that is so complete and can be used on-premise without extra costs I'm pretty sure it has saved a lot of money for us. We use FusionAuth on our own hosting and we're managing it on our own so the costs are the hosting cost plus our working hours' cost. After the initial setup, there is not much to do with FusionAuth except upgrading from time to time, so this way it is way more cost-effective than the competing solutions that are sometimes extremely expensive when it comes to a bigger number of users.
+
+**Dan:** How do you run FusionAuth (k8s, standalone tomcat server, behind a proxy, etc)?
+
+**Maciej:** We use the standalone tomcat server. Our production instances (and staging too as these are closely following production) are built to be highly available so there are two FusionAuth instances in a cluster, running behind a load balancer (Nova ADC) and using ElasticSearch cluster (3 nodes) with Postgresql database (managed, the highly available solution from the hosting provider).
+
+> Last but not least is that at Orbitvu we love open source and we use it a lot. What we do like in FusionAuth is that, while not open source, it feels like an open source project due to its development style with Github’s issue tracker.
+
+**Dan:** Any general feedback/areas to improve?
+
+**Maciej:** I'm not sure if there is anything to improve... maybe you can release less often so that I can stop upgrading my instances every week - just kidding, but the frequency of FusionAuth releases is amazing.
+
+One thing I'd like to emphasize is the support you have. I remember when I started using FusionAuth and the Forum section was empty. I don't know if it was due to some kind of forum migration or it has really started from scratch but after few weeks I've noticed with amazement how fast the new topics appear, how popular the platform is, and how good support you have (as said before, like an open source project).
```

Comment:
~I don't know if it was due to some kind of forum migration or it has really started from scratch but~ After **a** few weeks I ~'ve~ **stared noticing** ~with amazement~ how fast ~the~ new topics **were** appear **ing,** how popular the platform ~is~ **was,** and how good **the** support ~you have~ **was** (as said before, like an open source project).